lib/cc/analyzer/filesystem.rb

```diff
@@ -47,6 +37,12 @@ def files_matching(globs)
 
       private
 
+      def file_paths
+        @file_paths ||= Dir.chdir(@root) do
+          `find . -type f`.strip.split("\n")
```

Comment:
You've also now opened yourself up to funny filenaming issues. I'd suggest `-print0` and `split("\0")`.